presctl/README.MD

```diff
@@ -0,0 +1,30 @@
+# Presctl - Presidio's CLI utility
+
+## Installation
+1) Navigate to the presctl directory
+2) Run 'make' and the presctl binary will be created
+
+## Usage
+### prerequisites
+1. Running presidio cluster, either locally or remotely
+2. create a configuration file named '.presctl.yaml', place it in $HOME/.presctl.yaml and populate the required values [example file](example.presctl.yaml)
+
+### Instructions
+Simply running presctl will output the usage instructions
+```bash
+$ ./presctl
+```
+
+![usage output](./docs/images/usage-output.png)
+
+#### Examples
+
+1. Adding a new template 
```

Comment:
If you are providing an example files, it would also be nice to include the `template_def.json` as well

Reply:
we have multiple examples under the tester directory. I will refer the users there

Reply:
done